doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -18,11 +18,7 @@ Validated Versions
 ******************
 
 - Ubuntu version: **18.04**
-- ACRN hypervisor tag: **v2.6**
-
-.. note:: After the v2.6 release, the ``logical_partition`` scenario
-   was renamed to ``partitioned``, affecting the file names used in this
-   guide (validated with v2.6).
+- ACRN hypervisor commit: **9bae63b941**
```

Comment:
@NanlinXie @jackwhich I presume this is a temporary commit SHA and we'll update this here (and later in the document) to be v2.7?